modules/installation-vsphere-zones-prerequisites.adoc

```diff
@@ -0,0 +1,20 @@
+[id="installation-vsphere-zones-prerequisites_{context}"]
+== Prerequisites Specific to Multiple Failure Domains
+
+* All failure domains must share a common layer 3 network
+* Tag categories `openshift-region` and `openshift-zone` are created in vCenter.
+* Tags are applied to datacenters and compute clusters representing the name of their associated region and/or zone. 
```

Comment:
```suggestion
* Datacenters and compute clusters have tags representing the name of their associated region and/or zone. 
```
rewording to avoid passive voice